### ISSUE TITLE
Disabled text inputs don't have disabled border in HC Black theme

### ISSUE THREAD

**Issue by craxal** [microsoft/AzureStorageExplorer#2885]
**Storage Explorer Version**: 1.13.0
**Build Number**: dev
**Platform/OS**: All
**Architecture**: All
**Regression From**:  N/A

## Bug Description ##
When disabled in High Contrast Black theme, text inputs have a lime green color for text, but their borders are still white. This makes it harder to visually determine if a text input is enabled or disabled, especially if the input does not contain text.

## Steps to Reproduce ##
N/A

## Expected Experience ##
Disabled text boxes should have a lime green border.

## Actual Experience ##
Disabled text boxes have a white border.
